Library/Homebrew/tab.rb

```diff
@@ -250,6 +250,11 @@ def to_s
     when true  then s << "Poured from bottle"
     when false then s << "Built from source"
     end
+    unless time.nil?
+      s << "Installed" if s.empty?
```

Comment:
It feels like it might be nicer to have the `s << "Installed" if s.empty?` in a single place rather than two.